.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Test jdk15on
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Provide the branch to build and run tests on
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4.1.7
+      with:
+        ref: "${{ github.event.inputs.version }}"
+
+    - name: Set up JDK 8
+      uses: actions/setup-java@v4.2.1
+      with:
+        java-version: '8'
+        distribution: 'temurin'
+
+    - name: Set up Ant
+      uses: cedx/setup-ant@v2
+      with:
+        version: latest
+
+    - name: Verify Java version
+      run: java -version
+
+    - name: Set JDKPATH required for build
+      run: echo "JDKPATH=$JAVA_HOME" >> $GITHUB_ENV
+
+    - name: Run build script
+      run: bash ./build15+
+
+    - name: Upload Artifacts
+      uses: actions/upload-artifact@v4
+      with:
+        name: build-artifacts
+        path: build/artifacts/jdk1.5/jars/*.jar
+        retention-days: 5
```

Comment:
Upload all the jar files that are created in the output directory to Artifacts of the Workflow run
They will be retained for 5 days